cast_test.go

```diff
@@ -786,6 +786,38 @@ func TestToStringMapE(t *testing.T) {
 	}
 }
 
+func TestToStringMapStructE(t *testing.T) {
+	tests := []struct {
+		input  interface{}
+		expect map[string]struct{}
+		iserr  bool
+	}{
+		{map[string]interface{}{"v1": true, "v2": false}, map[string]struct{}{"v1": struct{}{}, "v2": struct{}{}}, false},
```

Comment:
I understand that the casting that this library does implies some conversion, but how can both true and false be the same as `struct{}{}`?